Makefile

```
@@ -87,20 +87,33 @@ lint: ## Perform static code analysis (check) using Black and Ruff
 e2e-test: ## Run end-to-end tests
 	@echo "Running end-to-end tests..."
-	@$(PYTHON) -m pytest tests/e2e -s
\ No newline at end of file
+	@$(PYTHON) -m pytest tests/e2e -s
+
+# ==============================================================================
+# CLEANUP
+# ==============================================================================
+
+.PHONY: clean
+clean: ## Remove __pycache__ and .venv to make project lightweight
```

checkmake (0.2.2)

[warning] Target body for "clean" exceeds allowed length of 5 (6).

(maxbodylength)